js/ui/components/discoveryFeed.js

```diff
@@ -0,0 +1,49 @@
+// -*- mode: js; js-indent-level: 4; indent-tabs-mode: nil -*-
+
+const Lang = imports.lang;
+
+const Main = imports.ui.main;
+const SideComponent = imports.ui.sideComponent;
+
+const DISCOVERY_FEED_NAME = 'com.endlessm.DiscoveryFeed';
+const DISCOVERY_FEED_PATH = '/com/endlessm/DiscoveryFeed';
+
+const DiscoveryFeedIface = '<node> \
+<interface name="' + DISCOVERY_FEED_NAME + '"> \
```

Comment:
For consistency here, I'd add a `const DISCOVERY_FEED_IFACE` that explicitly specifies the interface name. Some of the trouble I was debugging last night was because of confusion between the dbus name and the interface